stable/jenkins/values.yaml

```diff
@@ -470,6 +470,31 @@ agent:
   # Timeout in seconds for an agent to be online
   slaveConnectTimeout: 100
 
+  # Below is the implementation of custom pod templates for the default configured kubernetes cloud.
+  # Add a key under podTemplates for each pod template. Each key (prior to | character) is just a label, and can be any value.
+  # Keys are only used to give the pod template a meaningful name.  The only restriction is they may only contain RFC 1123 \ DNS label
+  # characters: lowercase letters, numbers, and hyphens. Each pod template can contain multiple containers.
+  # For this pod templates configuration to be loaded the following values must be set:
+  # master.JCasC.enabled: true
+  # master.JCasC.defaultConfig: true
+  # master.JCasC.configScripts: at least one key (cannot be empty {})
+  # Best reference is https://<jenkins_url>/configuration-as-code/reference#Cloud-kubernetes. The example below creates a python pod template.
+  podTemplates: {}
+  #  python: |
```

Comment:
I just noticed the JenkinsX [values](https://github.com/jenkins-x/jenkins-x-platform/blob/731a37428e057501e294d2737ebff785052e1fcf/jenkins-x-platform/values.yaml#L1057) doesn't make use of multi-line strings. If it's not truly needed then I'd vote to just make a toYaml call in the template.